helm/reana/values.yaml

```diff
@@ -56,15 +56,15 @@ components:
     enabled: true
   reana_server:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-server:0.7.0-alpha.1
+    image: reanahub/reana-server:0.7.0-alpha.2
```

Comment:
When updating this PR, you can edit `NOTES.txt` to put `mysecretpassword` back where `mypass1234` now is... I had shortened it to fit 80 columns, but the formatting now allows the original value `mysecretpassword` that we also use elsewhere... so I think we can put it back.